runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -297,6 +297,8 @@
 	 */
 	SpritePixels createItemSprite(int itemId, int quantity, int border, int shadowColor, int stackable, boolean noted, int scale);
 
```

Comment:
I would probably add documentation here similar to how the Player getLocalPlayer() is documented (no params just a return value). Since there are no parameters you'd probably only need @return.  I'd reccomend something along the lines of:

/**
* Creates an item...
*
* @return the created item
*/

Although it's pretty self explanatory, I'd probably just add it for consistency sake.